packages/date-picker/src/FilterDateRangePicker/FilterDateRangePicker.module.scss

```diff
@@ -0,0 +1,3 @@
+.filterDateRangePickerContainer {
+  display: contents;
```

Comment:
Hmm...In my case, the div is really just an invisible container because I wanted to put a data-testid on it for a story 😂 If I'm reading it right, the concern is that the semantics of the children may disappear? Or is it only for the container, in which case, it really is meant to not really be there. If I was to change it to not display contents, then I'd have to put extra styles on it and the children to ensure that the children still span the full width of the container. Not a lot of work, but since the intent is for it to simulate not existing, maybe it's acceptable to keep?